### ISSUE TITLE
Native logging is broken with ART 1.0.1

### ISSUE THREAD

**Issue by delpart** [IBM/adversarial-robustness-toolbox#189]
**Describe the bug**
The native python logger is not behaving as expected after import the adversarial robustness toolbox.

**To Reproduce**
Open a Python 3.7.4 interpreter and execute:
```
Python 3.7.4 (default, Oct  4 2019, 06:57:26) 
[GCC 9.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import logging
>>> import art
>>> logging.getLogger().setLevel(logging.INFO)
>>> logging.info('test')
>>> 
```

**Expected behaviour**
Logging should behave as expected, after importing art:
```
>>> import logging
>>> import art
>>> logging.getLogger().setLevel(logging.INFO)
>>> logging.info('test')
INFO:root:test
>>> 
```

**System information (please complete the following information):**
 - OS: Linux 5.3.7-arch1-1-ARCH
 - Python version: 3.7.4
 - ART version or commit number: 1.0.1 (installed through pip)
 - TensorFlow: 1.15.0rc2
 - Keras: 2.3.1

**Comment by ririnicolae** [comment]
Hi, @delpart! Thanks for your interest in ART. I am looking into the issue and will get back to you.

**Comment by ririnicolae** [comment]
Hey, @delpart! I was able to find the issue and fix it (see PR #190). This will be fixed on `pip` with the next release. Thanks again for bringing the bug to our attention!

**Comment by delpart** [comment]
@ririnicolae Thank you for your prompt response, looking forward to the next release.
